stories/screen.js

```diff
@@ -200,6 +200,7 @@ store.payments = [...Array(10)].map(() => ({
   status: 'complete',
   date: new Date(),
 }));
+store.payment.note = '#craefulgang';
```

Comment:
Added this so that the storybook shows a note

Reply:
:)